.github/workflows/cherry-pick-wp-release.yml

```diff
@@ -122,12 +123,32 @@ jobs:
               with:
                   script: |
                       const prNumber = context.issue.number;
+                      const commitSha = process.env.commit_sha;
                       const targetBranch = `wp/${process.env.version}`;
                       console.log(`prNumber: ${prNumber}`);
                       console.log(`targetBranch: ${targetBranch}`);
                       await github.rest.issues.createComment({
                         owner: context.repo.owner,
                         repo: context.repo.repo,
                         issue_number: prNumber,
-                        body: `There was a conflict while trying to cherry-pick the commit to the ${targetBranch} branch. Please resolve the conflict manually and create a PR to the ${targetBranch} branch.`
+                        body: `There was a conflict while trying to cherry-pick the commit to the ${targetBranch} branch. Please resolve the conflict manually and create a PR to the ${targetBranch} branch.
+
+                        PRs to ${targetBranch} are similar to PRs to trunk, but you should base your PR on the ${targetBranch} branch instead of trunk.
+
+                        \`\`\`
+                        # Checkout the ${targetBranch} branch instead of trunk.
+                        git checkout ${targetBranch}
+                        # Create a new branch for your PR.
+                        git checkout -b my-branch
+                        # Cherry-pick the commit.
+                        git cherry-pick ${commitSha}
+                        # Resolve the conflict...
+                        git add .
```

Comment:
Should we mention the contributor should make sure they don't have any unintended file changes? `git add .` will stage all files, so any that are not explicitly `.gitignore`d will be added.

Reply:
Sure, can add a line suggesting to check that with `git status` 👍